geowombat/ml/classifiers.py

```diff
@@ -344,6 +361,23 @@ def predict(
             .transpose("time", "band", "y", "x")
         )
 
+        # except AttributeError as e:
```

Comment:
Any reason for keeping this?

Reply:
No I will move this to a feature branch. Trying to figure out how to pass best_model from trained pipeline.